<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lab_hoagle">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hoàng Lê</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_ok">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OKAY</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_com">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="box_baud">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lab_com">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CỔNG COM</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_baud">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BAUDRATE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_setup">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SETUP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
